processing/src/main/java/io/druid/segment/data/ColumnarDoubles.java

```diff
@@ -54,6 +55,44 @@ public double getDouble(int offset)
         return ColumnarDoubles.this.get(offset);
       }
 
+      @Override
+      public boolean isNull()
+      {
+        return false;
+      }
+
+      @Override
+      public void inspectRuntimeShape(RuntimeShapeInspector inspector)
+      {
+        inspector.visit("columnar", ColumnarDoubles.this);
+        inspector.visit("offset", offset);
+      }
+    }
+    return new HistoricalDoubleColumnSelector();
+  }
+
+  default ColumnValueSelector<Double> makeColumnValueSelector(ReadableOffset offset, ImmutableBitmap nullValueBitmap)
+  {
+    class HistoricalDoubleColumnSelector implements DoubleColumnSelector, HistoricalColumnSelector<Double>
+    {
+      @Override
+      public double getDouble()
+      {
+        return ColumnarDoubles.this.get(offset.getOffset());
+      }
+
+      @Override
+      public double getDouble(int offset)
+      {
+        return ColumnarDoubles.this.get(offset);
+      }
+
+      @Override
+      public boolean isNull()
+      {
+        return nullValueBitmap.get(offset.getOffset());
```

Comment:
Should be inspected. Same for long and float.

Reply:
done.